### ISSUE TITLE
Loading MangaOCR manually

### ISSUE THREAD

**Issue by willyiam84** [bluaxees/Cloe#21]
Due to my network problem, I couldn't load MangaOCR successfully. So I want to know if I could download that model from any source and load it manually.

**Comment by blueaxis** [comment]
I believe [these](https://huggingface.co/kha-white/manga-ocr-base/tree/main) are the files that are downloaded when loading the model. Though I'm not sure how to manually place it in your machine. Can you paste here the error you're getting? It should be in your notifications

**Comment by ThoughtfulSenpai** [comment]
Not the original creator of this ticket but, as for notifications, this is pretty much all I got when opening Cloe without internet connection. 
![image](https://user-images.githubusercontent.com/100229664/231277534-d4809bb9-76e6-4c5c-9cd5-f000d5634380.png)

Pretty straightforward but not very informative. 
Anyway. So I cloned the [repo from huggingface ](https://huggingface.co/kha-white/manga-ocr-base/tree/main) , and a couple questions arise. 

1) Are 400 mb downloaded every time I open the app. That's pretty wasteful. I mean I have a solid internet connection and all but why download 400mb everytime? Or are we just accessing the model online? Since Poricom can function with the model deployed locally, is there an architectural reason why Cloe shouldn't be able to run 100% locally?

In any case, since I got the repo cloned, I would like to see if you have any clue whatsoever on how to set those files locall to the app folder and get rid of the requirement for internet, if possible. Any pointers or help would be appreciated enormously. The app is great, works flawlessly and it is an absolute charm to use, so I really would like to check if it would be possible to run it offline 100%. HAve a great day!

**Comment by blueaxis** [comment]
@ThoughtfulSenpai 

> Are 400 mb downloaded every time I open the app

No, it is downloaded once into your file system once. The reason why you need internet on the next load is because huggingface verifies whether you have the correct files in your system.

> Since Poricom can function with the model deployed locally, is there an architectural reason why Cloe shouldn't be able to run 100% locally?

No reason from the engineering side at all. I was busy these past few months so I just haven't added it yet.

> I would like to see if you have any clue whatsoever on how to set those files local to the app folder and get rid of the requirement for internet

Cloe and Poricom's codebase is a lot similar so you may check this [merge commit](https://github.com/blueaxis/Poricom/commit/a843dfb973fdb2dd92490de45456a1c2d3d250bd) and implement it yourself while waiting for the next release.

**Comment by AnarchystBR** [comment]
I'd very much like to run Cloe 100% offline too.
Of all the Capture2Text alternatives out there this is by far the best. No bells and whistles, just straight to the point.
Looking forward to the next release.

**Comment by Oshibuki** [comment]
+1

**Comment by AnarchystBR** [comment]
Receiving this today.
My Internet connection is ok.
Please make Chloe run 100% offline 🙏

![20240422103210](https://github.com/blueaxis/Cloe/assets/90112433/b0aa8667-2b75-41c0-b862-3b405a206a09)
